utils/techStack.ts

```diff
@@ -6,6 +6,8 @@ const techStack = [
   "python",
   "mongodb",
   "typescript",
+  "express",
```

Comment:
rename to `expressjs`
```suggestion
  "expressjs",
```